components/cam/bld/configure

```diff
@@ -1278,6 +1305,37 @@ EOF
 
 if ($print>=2) { print "Produce IOP file for SCAM: $e3smreplay$eol"; }
 
+#-----------------------------------------------------------------------------------------------
+# Superparameterization mode (MMF)
+
+if (defined $opts{'use_MMF'}) {
+    if (defined $opts{'use_ECPP'})            { $cfg_ref->set('use_ECPP', 1); }
+    if (defined $opts{'use_MAML'})            { $cfg_ref->set('use_MAML', 1); }
+    $cfg_ref->set('use_MMF', 1);
+    $cfg_ref->set('crm_nx', $opts{'crm_nx'});
+    $cfg_ref->set('crm_ny', $opts{'crm_ny'});
+    $cfg_ref->set('crm_nz', $opts{'crm_nz'});
+    $cfg_ref->set('crm_dx', $opts{'crm_dx'});
+    $cfg_ref->set('crm_dt', $opts{'crm_dt'});
+    if (defined $opts{'crm_nx_rad'}) { $cfg_ref->set('crm_nx_rad', $opts{'crm_nx_rad'}); }
+    if (defined $opts{'crm_ny_rad'}) { $cfg_ref->set('crm_ny_rad', $opts{'crm_ny_rad'}); }
+    $cfg_ref->set('MMF_microphysics_scheme', $opts{'MMF_microphysics_scheme'});
+    $cfg_ref->set('crm_adv', $opts{'crm_adv'});
+# Check that model is not using threading for the MMF 
+    if ($nthreads > 1) {
+     die <<"EOF";
+**  ERROR: MMF mode may only use single thread
+**         nthreads set to $nthreads
+**         
+EOF
+    }
+
+   # print "---> $opts{'crm_nx'} $opts{'crm_ny'} $opts{'crm_nz'} $opts{'crm_dx'} $opts{'crm_dt'} $opts{'MMF_microphysics_scheme'} $eol";
+}
+
+my $use_MMF = $cfg_ref->get('use_MMF') ? "ON" : "OFF";
```

Comment:
Why using "ON" or "OFF" for  use_MMF? 0 or 1 would be better. Plus, when $use_MMF="OFF", if ($use_MMF), which is used in the block start line 2660,  would still be true, causing unintended writing to src filepath.

Reply:
I missed something when not looking at the full file. use_MMF defined here would not be visible inside sub write_Filepath. In that case, use_MMF seems to be undefined in sub write_fileppath.

Reply:
Ben is fixing this too.